src/AspNet.Security.OAuth.GitHub/project.json

```diff
@@ -1,54 +0,0 @@
-﻿{
-  "version": "1.0.0-beta3-*",
-
-  "description": "ASP.NET Core security middleware enabling GitHub authentication.",
-  "authors": [ "Kévin Chalet", "Jerrie Pelser" ],
-
-  "packOptions": {
-    "owners": [ "Kévin Chalet", "Jerrie Pelser" ],
-
-    "projectUrl": "https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers",
-    "iconUrl": "https://avatars3.githubusercontent.com/u/7998081?s=64",
-    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0.html",
-
-    "repository": {
-      "type": "git",
-      "url": "git://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers"
-    },
-
-    "tags": [
-      "aspnetcore",
-      "authentication",
-      "github",
-      "oauth",
-      "security"
-    ]
-  },
-
-  "buildOptions": {
-    "warningsAsErrors": true,
-    "nowarn": [ "CS1591" ],
-    "xmlDoc": true
-  },
-
-  "dependencies": {
-    "AspNet.Security.OAuth.Extensions": { "target": "project", "type": "build" },
-    "JetBrains.Annotations": { "type": "build", "version": "10.1.4" },
-    "Microsoft.AspNetCore.Authentication.OAuth": "1.0.0"
-  },
-
-  "frameworks": {
-    "net451": { },
-
-    "netstandard1.3": {
-      "dependencies": {
-        "System.Dynamic.Runtime": "4.0.11"
```

Comment:
This dependency was necessary to make JSON.NET work with `dynamic`. It looks like it's no longer here in the .csproj. We'll have to make sure it doesn't cause a regression.